src/herder/HerderImpl.h

```diff
@@ -230,9 +230,10 @@ class HerderImpl : public Herder
     void writeDebugTxSet(LedgerCloseData const& lcd);
 
     ClassicTransactionQueue mTransactionQueue;
-    SorobanTransactionQueue mSorobanTransactionQueue;
+    std::shared_ptr<SorobanTransactionQueue> mSorobanTransactionQueue;
```

Comment:
Why not `unique_ptr` or `optional`? We don't intend to share the ownership for tx queues, do we?

Reply:
I think this was a remnant of some testing infra that got removed later. Updated to unique_ptr.